zap/src/main/java/org/parosproxy/paros/core/scanner/Scanner.java

```diff
@@ -569,4 +572,12 @@ public int getId() {
     public void setId(int id) {
         this.id = id;
     }
+
+    public Set<ScanFilter> getScanFilters() {
```

Comment:
This should have just package visibility (for now it's just for `HostProcess` to use).